code/game/objects/items/weapons/trays.dm

```diff
@@ -178,162 +168,153 @@
 */
 
 //Clicking an item individually loads it. clicking a table places the tray on it safely
-/obj/item/tray/afterattack(atom/target, mob/user as mob, proximity)
-	if (proximity)
-		if (istype(target, /obj/item))
+/obj/item/tray/afterattack(atom/target, mob/user, proximity)
+	if(proximity)
+		if(istype(target, /obj/item))
 			var/obj/item/I = target
 			attempt_load_item(I,user,1)
 
-	if (istype(target,/obj/structure/table))
-		safedrop = 1
-
+	if(istype(target,/obj/structure/table))
+		safedrop = TRUE
 
 //Alt+click with the tray in hand attempts to grab everything on the tile
 /obj/item/tray/alt_attack(var/atom/A, var/mob/user)
 	var/dist
 	var/tile
-	if (istype(A,/turf))
-		dist = get_dist(A,user.loc)
+	if(istype(A, /turf))
+		dist = get_dist(A, get_turf(user))
 		tile = A
 	else
-		dist = get_dist(A.loc,user.loc)
-		tile = A.loc
+		dist = get_dist(get_turf(A), get_turf(user))
+		tile = get_turf(A)
 
-	if (dist == 1)//checking that we're adjacent
+	if(dist == 1)//checking that we're adjacent
 		var/addedSomething = 0
 		for(var/obj/item/I in tile)
-			if (attempt_load_item(I, usr,0))
+			if(attempt_load_item(I, usr, 0))
 				addedSomething++
-		if ( addedSomething == 1)
-			usr.visible_message("<span class='notice'>[user] loads an item onto their service tray.</span>")
-		else if ( addedSomething )
-			usr.visible_message("<span class='notice'>[user] loads [addedSomething] items onto their service tray.</span>")
+		if(addedSomething == 1)
+			user.visible_message(SPAN_NOTICE("\The [user] loads an item onto their service tray."))
+		else if (addedSomething)
+			user.visible_message(SPAN_NOTICE("\The [user] loads [addedSomething] items onto their service tray."))
 		else
-			to_chat(user, "The tray is full or there's nothing valid here")
-			return 1
-		return 0//This prevents the alt-click from doing any farther actions
-	return 1
+			to_chat(user, SPAN_WARNING("The tray is full or there's nothing valid here."))
+			return TRUE
+		return FALSE //This prevents the alt-click from doing any farther actions
```

Comment:
god i know it's nitpicky and not something you added but this is driving me crazy
```suggestion
		return FALSE //This prevents the alt-click from doing any further actions
```

Reply:
father......